typescript/packages/plugins/renzo/src/parameters.ts

```diff
@@ -0,0 +1,36 @@
+import { createToolParameters } from "@goat-sdk/core";
+import { z } from "zod";
+
+export class DepositParams extends createToolParameters(
+    z.object({
+        _token: z.string().describe("The token to deposit"),
+        _amountIn: z.string().describe("The amount of tokens to deposit"),
+        _minOut: z.string().describe("The minimum amount of tokens to receive"),
+        _deadline: z.string().describe("The deadline for the deposit"),
```

Comment:
Which format does this follow? Check how we did it here in the KIM plugin: 
* https://github.com/goat-sdk/goat/blob/main/typescript/packages/plugins/kim/src/parameters.ts#L38
* https://github.com/goat-sdk/goat/blob/main/typescript/packages/plugins/kim/src/kim.service.ts#L48

Reply:
Implemented the first one. Much easier I felt